### ISSUE TITLE
Terraform Cloud Setup - Terraform Version and Remote Execution Mode

### ISSUE THREAD

**Issue by benemon** [hashicorp/field-workshops-terraform#352]
When prompted to undertake the Terraform Cloud setup tasks in both the AWS / Azure tracks (and probably GCP too, haven't checked), the instructions prompt the delegate tp execute `terraform version`, then navigate to `Settings -> General` to both set the Terraform Version in Terraform Cloud, and set the Remote Execution Mode to `Local` instead of `Remote`.

However, if you set the Remote Execution Mode to `Local`, the option to set the Terraform Version in Terraform Cloud is removed.

This presents an inconsistent user experience between the Instruqt Track and Terraform Cloud. It could potentially be confusing for delegates who haven't encountered Terraform Cloud before.

**Comment by gcastill0** [comment]
Yes. This is noted and captured in our recent audit work for the workshops. We are looking to optimize the work to support a more clear workflow.
